app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,11 @@
     <activity
       android:name="com.alexstyl.themingandroid.advanced.AdvancedExampleActivity"
       android:label="@string/Advanced_Example"
-      android:parentActivityName="com.alexstyl.themingandroid.simple.SimpleExampleActivity" />
+      android:parentActivityName="com.alexstyl.themingandroid.simple.SimpleExampleActivity">
+    <meta-data
+        android:name="android.support.PARENT_ACTIVITY"
+        android:value="com.alexstyl.themingandroid.simple.SimpleExampleActivity"/>
+    </activity>
```

Comment:
No particular reason for downgrading to 14. I just needed to use this code for my 2 apps that have minSdkVersion 14 and minSdkVersion 9. I can switch it back to 16 if you want and delete the `<meta>` instructions since it's not needed in sdk >= 16